.bit/responses/2.2-Week 2 Step 2.md

```diff
@@ -116,13 +116,13 @@ In the `multipart.Parse()` call, you need to figure out what the body parameter
 ```js
 
 // here's your boundary:
-var boundary = multipart.getBoundary(req.headers['content-type']);
+const boundary = multipart.getBoundary(req.headers['content-type']);
   
 // TODO: assign the body variable the correct value
-var body = '<WHAT GOES HERE?>'
+let body = '<WHAT GOES HERE?>'
```

Comment:
are we manipulating the `body` variable later? if not can we change it to a let to keep things consistent?

Reply:
done

Reply:
looks awesome 🙂